File: sketches/java/FizzBuzz/src/Main/Format_multiples_of_3.java
package Main;

public class Format_multiples_of_3 implements INumberFormatter {

	@Override
	public boolean can_handle(int number) {
		
		return number%3==0;
	}

	@Override
	public String handle(int number) {
		
		return "Fizz";
	}
 
}

File: sketches/spring/HelloSpring/HelloSpring/FormatMultiplesOf3.cs
namespace HelloSpring
{
    public class FormatMultiplesOf3 : INumberFormatter
    {
        public bool CanHandle(int number)
        {
            return number%3 == 0;
        }

        public string Handle(int number)
        {
            return "Fizz";
        }
    }
}